SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetBitReader.cpp

```diff
@@ -9,69 +9,75 @@
 
 DEFINE_LOG_CATEGORY(LogSpatialNetBitReader);
 
+static thread_local FSpatialNetBitReader* s_CurrentReader = nullptr;
```

Comment:
Is this `s_` an established code style?

Reply:
Only in a world where you prefix variable with a hint to their lifecycle scope :(